### PR TITLE
potential fix for crd with empty cut

### PR DESCRIPTION
**Pull request by MengLiuPurdue**

### CODE REVIEW

File: localgraphclustering/src/lib/capacity_releasing_diffusion.cpp [hunk 1]
```diff
@@ -240,8 +245,13 @@ void graph<vtype,itype>::round_unit_flow(unordered_map<vtype,vtype>& l,
     	vtype i = iter->first;
     	double _a = vol[i];
     	double _b = (volume - _a);
+        //cout << "i: " << i << " " << "a: " << _a << " " << "b: " << _b << endl;
     	double demominator = _a < _b ? _a : _b;
+        if (iter->second == 0) {
```
[the remark this answers is not in the document]
Comment (MengLiuPurdue) [reply localgraphclustering/src/lib/capacity_releasing_diffusion.cpp]: From my understanding, the denominator can be 0 only if the numerator is zero as well.

**Comment by MengLiuPurdue** [comment]
@dgleich Let me know how you think on my latest commit.

**Comment by dgleich** [comment]
Meng, it stilll seems like we can improve this one :)

Keep working on it.

**Comment by dgleich** [comment]
Meng, let's do this -- switch the stages to include two types of tests.

The first type is one that impacts coverage. Primarily this should focus on the python library as that's where we get coverage info :)

The second type doesn't have any coverage info. This just deeply tests algs like CRD. So we only care if the second type passes or not. So coveralls shoudl only run on the first type.

Make sense

**Comment by MengLiuPurdue** [comment]
> Meng, let's do this -- switch the stages to include two types of tests.
> 
> The first type is one that impacts coverage. Primarily this should focus on the python library as that's where we get coverage info :)
> 
> The second type doesn't have any coverage info. This just deeply tests algs like CRD. So we only care if the second type passes or not. So coveralls shoudl only run on the first type.
> 
> Make sense

Sounds good.

**Pull request merged**